x11-packages/dosbox-x/build.sh

```diff
@@ -0,0 +1,18 @@
+TERMUX_PKG_HOMEPAGE=https://dosbox-x.com/
+TERMUX_PKG_DESCRIPTION="DOSBox-X fork of the DOSBox project"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=0.83.22
+TERMUX_PKG_REVISION=1
+TERMUX_PKG_SRCURL=https://github.com/joncampbell123/dosbox-x/archive/refs/tags/dosbox-x-v${TERMUX_PKG_VERSION/.3/-3}.tar.gz
+TERMUX_PKG_SHA256=
+TERMUX_PKG_DEPENDS="libc++, libpng, libiconv, libx11, sdl2, sdl2-net, zlib"
+
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
+--disable-avcodec
+--enable-sdl2
+--disable-dynamic-x86
+--disable-fpu-x86
+--disable-opengl
+--prefix=/usr "$@"
```

Comment:
Remove `--prefix=/usr`. It will be passed by configure step.

Also, there is no need of `$@`.

Reply:
You should read [Creating-new-package](https://github.com/termux/termux-packages/wiki/Creating-new-package) before making PR.